content/applications/sales/sales/invoicing/expense.rst

```diff
@@ -155,6 +155,34 @@ Upon clicking :guilabel:`Post Journal Entries`, that button disappears, and the
 Distribution` column in the :guilabel:`Expense` tab is filled with the sales order that was
 initially configured to the expense in the :guilabel:`Customer to Reinvoice` field.
 
+.. important::
+   By default, the :guilabel:`Customer to Reinvoice` field is enabled for the :guilabel:`[TRANS &
+   ACC] Travel & Accomodation` expense category.
```

Comment:
Checking: is this the *only* expense category it's enabled for?

Reply:
After looking into this, it appears that this field also appears for: [COMM] Communication, [FOOD] Meals, [MIL] Mileage, and it's not available for [EXP_GEN] Others or [GIFT] Gifts. Making that adjustment now. Thanks for bringing this to my attention.